README.md

```diff
@@ -22,7 +22,6 @@ MXNet, PyTorch, XGBoost, and others.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
```

Comment:
Can you add a line here, please ? 
Our script removes this line 😄 , we should modify it.

Reply:
Sure!

Reply:
Added it with the latest commit